violet-framework/src/main/java/com/horstmann/violet/workspace/sidebar/graphtools/GraphToolsBarPanel.java

```diff
@@ -111,7 +111,7 @@ private GraphToolsBarButton getSelectedButton()
 
 
 
-
+/*
```

Comment:
Why this fragment is commented out ? Is it your change ? You forgot to remove this ?